package tutorial.orcas.combat.server.event;

enum EventType {
  EVENT_TEST = 1;

  EVENT_COMBAT_BEGIN = 1001;
  EVENT_COMBAT_MOVE_ACTION = 1002;
  EVENT_COMBAT_BUILD_ACTION = 1003;
  EVENT_COMBAT_LOCK_TARGET = 1004;
  EVENT_COMBAT_DEATH = 1005;
  EVENT_COMBAT_CONVERT_CAMP = 1006;
  EVENT_COMBAT_ATTACK = 1007;
  EVENT_COMBAT_ATTACKED = 1008;
  EVENT_COMBAT_SETTLE = 1009;

  EVENT_SCENE_MOVE_STEP = 2001;
}
